rust_wtx_bench/Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM docker.io/library/debian:bookworm-slim AS build
+WORKDIR /app
+COPY rust_wtx_bench /app
+COPY proto /app/protos
+RUN apt-get update && apt-get install curl gcc -y
+RUN curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh -s -- --default-toolchain nightly-2024-07-10 --profile minimal -y
+ENV CARGO_PROFILE_RELEASE_LTO=true
+RUN RUSTFLAGS="-Ccodegen-units=1 -Copt-level=3 -Cpanic=abort -Cstrip=symbols -Ctarget-cpu=native" $HOME/.cargo/bin/cargo build --release
```

Comment:
Let's have them in Cargo file, otherwise they're present only in the container.

Reply:
Done